_data/settings/components/accordion.yml

```
@@ -10,6 +15,11 @@ contents:
     var: $theme-accordion-border-color
     default: '"base-lightest"'
     type: color
+  - name: Button background color
+    description: Button background color of the accordion.
```

A possible tiny tweak:
```suggestion
    description: Background color of the accordion button.
```